docker_push.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+VERSION="0.29"
+
+docker build -t costimuraru/ops-cli:${VERSION} .
+
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
```

Comment:
@costimuraru we need to set these two env var in Travis for this to work.
Should we create a dedicated docker hub account for this?

Reply:
solved - moved to `adobe/ops-cli`